
# @ref VelocityPlanningResult
int8 tar_type

float32 tar_v

float32 tar_a

bool release_throttle

float32 tar_throttle

bool aeb_warning

bool aeb_action

int8 obs_valid

int8 obs_dec_status

float32 obs_dist_to_obj

float32 obs_time_gap

float32 obs_relative_v

float32 obs_abs_v

float32 obs_ttc

int8 obj_id

float32 mpu_cur_slope

float32 ADASV2_cur_slope


